### ISSUE TITLE
Pan and zoom should be persistent

### ISSUE THREAD

**Issue by schoofseggl** [n3-charts/line-chart#540]
When updating the data of a data series (say with an AJAX call), the zoom and pan is reset. This is rather inconvienient because a user has to repeat the zoom over and over again to see the section of interest if the data changes on a regular base. 

Is there a possibility to "pin" the zooming or to data bind the zoom factor?

**Comment by xlucasdemelo** [comment]
I have the same issue here.

**Comment by chaosmail** [comment]
Hi guys, sorry for the late reply!

Is this still an issue for you?

Did you try to set the `domain-sync-key` to sync the pan/zoom domain to a scope property?

Best,
Christoph

**Comment by schoofseggl** [comment]
No, I haven't tried it yet, I realized this not before I wrote this report ^^. Is there a short snippet anywhere in the documentation on how this could be done?
